src/index.jsx

```diff
@@ -27,6 +28,11 @@ import Head from './head/Head';
 
 import AppRoutes from './routes/AppRoutes';
 
+const RealFooter = () => {
```

Comment:
NIT: Maybe consider calling this `renderFooter` or something like that? In my opinion, it feels like "real" implies that there is a fake or mocked one as well

Reply:
Yup, makes sense!